src/main/java/com/yelp/nrtsearch/server/luceneserver/QueryNodeMapper.java

```diff
@@ -719,8 +720,19 @@ private static FuzzyQuery getFuzzyQuery(
         protoSpanMultiTermQuery.getFuzzyQuery();
     Term term = new Term(protoFuzzyQuery.getField(), protoFuzzyQuery.getText());
 
-    int maxEdits =
-        protoFuzzyQuery.hasMaxEdits() ? protoFuzzyQuery.getMaxEdits() : FuzzyQuery.defaultMaxEdits;
+    int maxEdits = FuzzyQuery.defaultMaxEdits;
+    if (protoFuzzyQuery.hasAuto()) {
+      com.yelp.nrtsearch.server.grpc.FuzzyParams.AutoFuzziness autoFuziness =
+          FuzzyParams.AutoFuzziness.newBuilder()
+              .setLow(protoFuzzyQuery.getAuto().getLow())
+              .setHigh(protoFuzzyQuery.getAuto().getHigh())
+              .build();
+      maxEdits = computeMaxEditsFromTermLength(term, autoFuziness);
```

Comment:
Could you use the `AutoFuzziness` message from `FuzzyParams` directly to avoid this rebuild?

Reply:
Moved `AutoFuzziness` message from within `FuzzyParams` to a top level message.